packages/cmr-client/src/CMR.ts

```diff
@@ -36,20 +34,26 @@ const userIpAddress = () =>
  *
  * @private
  */
-async function updateToken(cmrProvider, clientId, username, password) {
-  // if (!cmrProvider) throw new Error('cmrProvider is required.');
-  // if (!clientId) throw new Error('clientId is required.');
-  // if (!username) throw new Error('username is required.');
-  // if (!password) throw new Error('password is required.');
+async function updateToken(
+  cmrProvider: string,
```

Comment:
So TS is now enforcing the existence of the arguments?

Reply:
It is, but that commented-out code definitely wasn't enforcing it before.